processing-tests/common/test/kotlin/com/livefront/sealedenum/testing/CompilationAssertions.kt

```diff
@@ -7,15 +7,15 @@ import org.junit.jupiter.api.Assertions.assertEquals
 /**
  * Asserts that the given [result] compiled successfully.
  */
-internal fun assertCompiles(result: KotlinCompilation.Result) {
```

Comment:
Are you able to remove the `KotlinCompilation` import?

Reply:
Negative, I'm still using the `KotlinCompilation.ExitCode.OK` below.